src/Altinn.Notifications/Validators/EmailNotificationOrderRequestValidator.cs

```diff
@@ -17,8 +17,8 @@ public class EmailNotificationOrderRequestValidator : AbstractValidator<EmailNot
     public EmailNotificationOrderRequestValidator()
     {
         RuleFor(order => order.Recipients)
-            .NotEmpty()
-            .WithMessage("One or more recipient is required.")
+            .Must(r => r?.Count is >= 1 and <= 50) // Azure Communication Services has max 10 recipients
```

Comment:
Number of recipients for an order doesn't directly translate to how many recipients are set in the to field of an email. One email is sent per recipient.

Reply:
Ahh, makes sense! Should there be still some upper bound? Like should [Array.MaxLength (2,147,483,591)](https://learn.microsoft.com/en-us/dotnet/api/system.array.maxlength?view=net-8.0) number of recipients be allowed?

Reply:
I would like to keep it open for now. We are still working on analyzing how batch orders should be handled and will most likely have a separate endpoint for posting extra large orders (think information about tax forms being available), once this has been specified we will be able to set a reasonable limit for this endpoint. Current use cases, e.g. notifications from an app triggered by the user or by instantiation will not generate any significant number of recipients is our current hypothesis.